sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -799,11 +801,7 @@ final void moveToOutputFiles(
       }
       // During a failure case, files may have been deleted in an earlier step. Thus
       // we ignore missing files here.
-      FileSystems.rename(
-          srcFiles,
-          dstFiles,
-          StandardMoveOptions.IGNORE_MISSING_FILES,
-          StandardMoveOptions.SKIP_IF_DESTINATION_EXISTS);
```

Comment:
This was set to "SKIP_IF_DESTINATION_EXISTS" to make sure that we do not fail when partially completed workitems are retried by the runner. I don't think making this customizable is a good option since passing different move options could result in hard to track (and inconsistent) retry behavior changes for users.